### DISCUSSION TITLE
reactivesearch dashboard.

### DISCUSSION THREAD

**Discussion by Reprovo** [appbaseio/reactivesearch#1598] [Q&A]
Hi. I am trying to setup the reactivesearch dashboard and have set the URL and APP constants to match my elasticsearch instance
but am receiving the following:

export const APP = 'logstash-ftd-test';

export const URL = 'http://testsite:9200';

http://testsite:9200/logstash-ftd-test/_reactivesearch.v3 has been clobkec by CORS policy.

I was wondering why the '_reactivesearch.v3' is being appended to the request.

**Comment by siddharthlatest** [comment]
@Reprovo What are you referring to as ReactiveSearch dashboard in this case? You want to ensure that `enableAppbase` prop in ReactiveBase is set to `false` to query Elasticsearch directly.
